src/basic/strv.c

```diff
@@ -242,13 +242,13 @@ int strv_extend_strv(char ***a, char * const *b, bool filter_duplicates) {
         return -ENOMEM;
 }
 
-int strv_extend_strv_append(char ***a, char * const *b, const char *suffix) {
+int strv_extend_strv_prepend_append(char ***a, const char *prefix, char * const *b, const char *suffix) {
```

Comment:
I mean, this is even more confusing... Can't this just be called `strv_extend_strv_concat_full` or something?